src/app/checkout/step-3/step-3.tpl.html

```diff
@@ -150,6 +150,22 @@
     </div>
   </div>
 
+  <div class="mb" ng-if="!!$ctrl.brandedCheckoutItem['premium-code']">
```

Comment:
Am I understanding this correctly that, if they chose to add a premium, this section will show up, but if they chose not to add a premium, this section will not show up?

Reply:
Yes, that is what I was thinking.  Although maybe we can render this section if it is branded checkout and if the branded checkout configuration has premium data; and if the premium was not selected, we could show the phrase "none" in this box?

Reply:
It should probably be asked in Jira, but I don't think it should hold up putting something on staging necessarily if we don't get an answer before fixing the other issues here.